sdk/@launchdarkly/observability-dotnet/AspSampleApp/Program.cs

```diff
@@ -0,0 +1,54 @@
+using LaunchDarkly.Observability;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddLaunchDarklyObservability(
+    Environment.GetEnvironmentVariable("LAUNCHDARKLY_SDK_KEY"),
+    ldBuilder =>
+    {
+        ldBuilder.WithServiceName("ryan-test-service");
+    }
+);
```

Comment:
how would the API change with the plugin spec alongside the launchdarkly FM SDK?

Reply:
The plugin is coming in a subsequent PR. It basically calls this function with the appropriate information. The plugin constructor itself will take the options and service collection. Not 100% solid on the API itself yet. 

Either:
```
ObservabilityPlugin.Builder(services).WithServiceName("ryan-test-service");
```

Or:
```
ObservabilityPlugin.Build(services, builder => {
  builder.WithServiceName("ryan-test-service");
});
```

I am leaning toward leaving this exposed though, for cases where DI makes it hard for people to make the LD SDK early enough.